AutoGVP/03-add_INFO_fields.R

```diff
@@ -0,0 +1,59 @@
+################################################################################
+# 03-add_INFO_fields.R
```

Comment:
I think I misspoke on numbering - or we need to make a decision. Either we remove the numbers altogether or make this 02. I just checked and didn't realize both initial scripts are 01. It is confusing to have two 01s. Maybe these just all get deleted for when they go into the bash run script?